src/app/components/progress-indicator/index.html

```diff
@@ -1,20 +1,266 @@
+			</ul>
+	</sky-demo-page-content>    
+    
+    </sky-tab>
+    
+    <sky-tab tabHeading="Passive indicator design">
+    
+    <sky-demo-page-content sectionHeading="Usage usage">
+	<h3 class="sky-text-success">Use when</h3>
+		<sky-docs-visual-example>
+			<sky-docs-visual-example-text>
+					<p>Use passive progress indicators when users must be aware of sequential progress that concerns them, but they are not responsible for.</p>
+			</sky-docs-visual-example-text>
+			<sky-docs-visual-example-image>
+				<div>
+					<img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-1.png" />
+					<div class="caption caption-success sky-text-success">Do use passive progress indicators to track sequential steps taken by other users in the system.</div>
+				</div>
+			</sky-docs-visual-example-image>
+		</sky-docs-visual-example>
+    
+    <h3 class="sky-text-danger">Don't use when</h3>
+        <sky-docs-visual-example>
+			<sky-docs-visual-example-text>
+					<p>Don't use passive progress indicators when users must interact with the steps.</p>
+			</sky-docs-visual-example-text>
+			<sky-docs-visual-example-image>
+				<div>
+					<img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-2.png" />
+					<div class="caption caption-danger sky-text-danger">Don't use passive progress when users must take action on one or more steps.</div>
+				</div>
+			</sky-docs-visual-example-image>
+		</sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+			<sky-docs-visual-example-text>
+					<p>Don’t use passive progress indicators to represent multiple independent statuses that are not sequentially related.</p>
+			</sky-docs-visual-example-text>
+			<sky-docs-visual-example-image>
+				<div>
+					<img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-3.png" />
+					<div class="caption caption-danger sky-text-danger">Don't use passive progress to track steps that have no sequence.</div>
+				</div>
+			</sky-docs-visual-example-image>
+		</sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+			<sky-docs-visual-example-text>
+					<p>Don't use multiple passive progress indicators at once on items in a list.</p>
+			</sky-docs-visual-example-text>
+			<sky-docs-visual-example-image>
+				<div class="sky-margin-stacked-separate">
+					<img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-4.png" />
+					<div class="caption caption-danger sky-text-danger">Don't expose multiple passive progress indicators at once.</div>
+				</div>
+                <div>
+					<img src="~/assets/img/guidelines/progress-indicator/passive-progress-usage-5.png" />
+					<div class="caption caption-success sky-text-success">Do roll up steps that are inconsequential into a single status. Use a container like Popover or Flyout to hold the additional details the passive progress indicator provides.</div>
```

Comment:
Should this show a popover holding the steps?

Reply:
Agreed. Updated the visual example.